client/src/components/Admin/Jobs/index.vue

```diff
@@ -1,5 +1,7 @@
 <template>
     <div>
+        <b-container>
+            <b-card> <b-button @click="createJob">Create Job</b-button></b-card></b-container>
```

Comment:
always remember to keep components near where you create them, place job component here